### DISCUSSION TITLE
[Enhancement]: A few suggestions for the convenience of the books.

### DISCUSSION THREAD

**Discussion by ra3ggi** [advplyr/audiobookshelf#1780] [Ideas]
### Describe the feature/enhancement

1. Possibility to merge several libraries into one.
2. Ability to transfer books from one library to another.
3. Possibility for administrator to delete books together with book files (irretrievable deletion).

**Comment by nschepsen** [comment]
+ up

any news on this suggestion?

**Comment by advplyr** [comment]
3 was added. Merging libraries and transferring books would be moving the actual files/folders in the file system.
That may be something added but the UX would need to be designed first

**Comment by DieselTech** [comment]
Wouldn't merging several libraries just be done in the edit library page by listing all the respective folders?

![image](https://github.com/advplyr/audiobookshelf/assets/30128380/4da0ca63-12f1-45e3-bcb1-cce213d75b6b)

**Reply by advplyr** [reply]
You can do it that way but the library items would be created as new. A new library item wouldn't retain user progress or updated metadata that isn't stored in the item folder or embedded.
